L1Trigger/L1THGCal/python/hgcalTriggerPrimitiveDigiProducer_cfi.py

```diff
@@ -67,9 +69,10 @@
                           minPt_multicluster = cms.double(0.5), # minimum pt of the multicluster (GeV)
                           calibSF_multicluster = cms.double(1.084),
                           type_multicluster = cms.string('dRC3d'), #'DBSCANC3d' for the DBSCAN algorithm 
+                          applyLayerCalibration = cms.bool(True),
+                          layerWeights = layercalibparam.AllLayer_weights,
```

Comment:
Is it intended that the AllLayer_weights are used by default here? (instead of the TrgLayer_weights)

Reply:
Here the layers have not been disconnected, so the calib is consistent with that.